src/loader/BlendLoader.py

```diff
@@ -16,38 +16,121 @@ class BlendLoader(LoaderInterface):
     information about a datablock see Blender's documentation for bpy.types.ID
     at https://docs.blender.org/api/current/bpy.types.ID.html
 
-    Sections/Datablocks in a .blend File
-
-    +-------------------+
-    |    .Blend File    |
-    +===================+
-    |    Collections    |
-    +-------------------+
-    |    Object         |
-    +-------------------+
-    |    Mesh           |
-    +-------------------+
-    |    Text           |
-    +-------------------+
-    |    Scene          |
-    +-------------------+
-    |    World          |
-    +-------------------+
-    |    Workspace      |
-    +-------------------+
-    |    Curve          |
-    +-------------------+
-    |    Camera         |
-    +-------------------+
-    |    Light          |
-    +-------------------+
-    |    Material       |
-    +-------------------+
-    |    Texture        |
-    +-------------------+
-    |    more...        |
-    +-------------------+
+    .. list-table:: 
+        :widths: 25 100 10
+        :header-rows: 1
 
+        * - Datablock
```

Comment:
Are we sure we want to support all of these data types?

I would not recommend for example using Brushes as they are only usable in the window mode. Same goes for CacheFile, FreestyleLineStyle, GreasePencil, WindowManager, WorkSpace.

For Library we need to be extra careful, I just learned that usually blender will try to link an existing library instead of appending it. The difference being that a linked library does not really count, we can iterate over the objects or change them in any way besides a global transformation on all objects in that linked library. So we won't be able to change materials or do anything else, which also prevents them from being used with the `SegMapRenderer`. So, we have to avoid at all cost that Library get linked, they *always*  have to be appended. 

Can you please make sure that, if the attribute Library is used that the objects are appended and not linked.

Reply:
> For Library we need to be extra careful, I just learned that usually blender will try to link an existing library instead of appending it.

From documentation it seems that bpy.ops.wm.append(... link=False) by default appends and links only when explicitly stated as per documentation
 - link (boolean, (optional)) – Link, Link the objects or data-blocks rather than appending

Reply:
> I would not recommend for example using Brushes as they are only usable in the window mode. Same goes for CacheFile, FreestyleLineStyle, GreasePencil, WindowManager, WorkSpace.

Should they be explicitly checked and prevented to be loaded? Currently user provided values are matched with the types supported by the API so all types supported are loaded without type checking for specific types, else we would be back to old key value pair check.